### PR TITLE
Feature dashboard pull secret

### PR DESCRIPTION
**Pull request by ProfessorMc**
# Description

_Please explain the changes you've made_

## Issue reference
#2475 
We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #2475 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: charts/dapr/README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | Parameter                                 | Description                                                             | Default                 |
 |-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
 | `global.registry`                         | Global Dapr docker image registry                                       | `docker.io/daprio`      |
-| `global.tag`                              | Global Dapr docker image version tag                                    | `0.11.0`                |
```
Comment (artursouza) [inline charts/dapr/README.md:66]: We are not publishing RC's as the latest version to be installed *by default*. Users might ask for it as a parameter.

File: docs/release_notes/v1.0.0-rc.1.md [hunk 2]
```diff
@@ -193,6 +193,8 @@ These are some highlights for this release;
 
 - **FIXED** Display app name (id) in Dashboard on K8s [87](https://github.com/dapr/dashboard/issues/87)
 
+- **ADDED** Support for imagePullSecret Secret on Helm Chart [2475](https://github.com/dapr/dapr/issues/2475)
```
Comment (artursouza) [inline docs/release_notes/v1.0.0-rc.1.md:196]: Thanks.

**Review by artursouza — CHANGES_REQUESTED** [review]
Just one small fix. Otherwise, LGTM.

**Comment by yaron2** [comment]
@ProfessorMc thanks for this PR. See @artursouza's comment.

**Comment by yaron2** [comment]
@ProfessorMc please see requested changes. once fixed, we can go ahead and merge this PR.

File: charts/dapr/README.md (hunk 1 above)
Comment (ProfessorMc) [reply charts/dapr/README.md:66]: @artursouza I see, but this was confusing.  The values.yaml file currently sets the default to "1.0.0-rc.1"  should that be rolled back?

Comment (yaron2) [reply charts/dapr/README.md:66]: You're right, that is confusing. We will rollback values.yaml to 0.11.3.

**Review by yaron2 — APPROVED** [review]

**Pull request merged**